Penteract/Assets/Scripts/RangedAI.cpp

```diff
@@ -71,6 +74,22 @@ void RangedAI::Start() {
 			noDmgMaterialID = meshRenderer->materialId;
 		}
 	}
+
+	int numChildren = GetOwner().GetChildren().size();
+	if (numChildren > 2) {
+		GameObject* weaponGO = GetOwner().GetChildren()[2];

```

Comment:
I know was done with other things already but I do not like this hardcoded children positions... At least we should leave somewhere written a warning of not to change children positions inside ranged prefab hierarchy

Reply:
You're definetly right. I will be adding an empty GameObject to Enemies saying the hierarchy should not be modified and will be replacing the hardcode values. Ty